docs/powerauth-server/PowerAuth-Server-1.3.0.md

```diff
@@ -106,6 +106,147 @@ CREATE UNIQUE INDEX pa_application_name_index
     ON pa_application_version (application_id, name);
 ```
 
+## Create Auditing Tables
+
+Create tables for auditing:
+- `audit_log` - table used to store audit logs
+- `audit_param` - table used to store detailed parameters for audit logs
+
+### Oracle
+
+```sql
+--
+-- Create audit log table.
+--
+CREATE TABLE audit_log (
+    audit_log_id       VARCHAR2(36 CHAR) PRIMARY KEY,
+    application_name   VARCHAR2(256 CHAR) NOT NULL,
+    audit_level        VARCHAR2(32 CHAR) NOT NULL,
+    audit_type         VARCHAR2(256 CHAR),
+    timestamp_created  TIMESTAMP,
+    message            CLOB NOT NULL,
+    exception_message  CLOB,
+    stack_trace        CLOB,
+    param              CLOB,
+    calling_class      VARCHAR2(256 CHAR) NOT NULL,
```

Comment:
Just wondering whether 256 chars is enough, package name + class + proxy marker... But I guess it is ok.

Reply:
We control the classes which call the auditing interface, it should be fine.